src/app-layer-ftp.c

```diff
@@ -477,30 +507,7 @@ static int FTPParseRequest(Flow *f, void *ftp_state,
 
 static int FTPParsePassiveResponse(Flow *f, FtpState *state, uint8_t *input, uint32_t input_len)
 {
-    uint16_t dyn_port;
-
-#ifdef HAVE_RUST
-    dyn_port = rs_ftp_pasv_response(input, input_len);
```

Comment:
This won't work in rust because the rust parsing function is looking for something starting with 227 response code.

Reply:
Sorry, i don't know rust, so can anybody help me to add a method rs_ftp_active_response in the rust file?

Reply:
You can skip the rust code and use some C code only if you refactor a bit. But rust is not that complicated if you have example.